src/webhooks/types/events/directory_user_updated.rs

```diff
@@ -0,0 +1,125 @@
+use std::collections::HashMap;
+
+use serde::Deserialize;
+use serde_json::Value;
+
+use crate::directory_sync::DirectoryUser;
+
+#[derive(Debug, PartialEq, Eq, Deserialize)]
+
+/// Type for a [`DirectoryUser`] with previous attributes object returned in a [`dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
+pub struct DirectoryUserWithPreviousAttributes {
+    #[serde(flatten)]
+    /// [`DirectoryUser`] that will be included in the [`DirectoryUserWithPreviousAttributes`]
+    pub directory_user: DirectoryUser,
+
+    /// Type for previous attributes that were updated in a [`dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
+    pub previous_attributes: HashMap<String, Value>,
+}
+/// [WorkOS Docs: `dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
```

Comment:
Could we add a blank line here?

```suggestion

/// [WorkOS Docs: `dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
```